File: PracaDomowa/CosmosDBPSW/CosmosDBPSW/Animal.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Microsoft.Azure.Cosmos.Table;

namespace CosmosDBPSW
{

    class Animal : TableEntity
    {

		public string Name { get; set; }
		public int BirthYear { get; set; }
		public string Comments { get; set; }
		public int Catwalk { get; set; }

		public string Species { get; set; }
		public string  Breed { get; set; }


		
		public Animal()
		{
			PartitionKey = "animals";
			RowKey = Guid.NewGuid().ToString();
		}

		override public string ToString()
		{
			return "Zwierze " + Name + " urodzone w " + BirthYear + ", Uwagi: " + Comments + ", nr wybiegu: " + Catwalk  + ", gatunek: " + Species + ", rasa: " + Breed;
		}
	}
}

File: Zajęcia/LabHazelcast/src/main/java/Animal.java
import java.io.Serializable;

public class Animal implements Serializable {

	private static final long serialVersionUID = 1L;

	public String name;
	public int birthyear;
	public String comments;
	public int catwalk;
	public String breed;
	public String species;


	public Animal(String name, int birthyear, String comments, int catwalk, String breed, String species)
	{
		this.name = name;
		this.birthyear = birthyear;
		this.comments = comments;
		this.catwalk = catwalk;
		this.breed = breed;
		this.species = species;

	}



	@Override
	public String toString(){
		return "Zwierze " + name + " urodzone w " + birthyear + ", Uwagi: " + comments + ", nr wybiegu: " + catwalk  + ", gatunek: " + species + ", rasa: " + breed;
	}


}
